ecpy_hqc_legacy/instruments/drivers/visa/rohde_and_schwarz_zva24.py

```diff
@@ -0,0 +1,993 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is this just a large copy-paste of the ZNB20 driver ? If so it would make more sense to simply declare the instrument in the manifest and use the same driver or subclass this driver from the ZNB20 to avoid duplicating so much code.

Reply:
Few SCPI commands differ between these instruments.